### PR TITLE
added threshold to g-api imgproc tests

### PR DESCRIPTION
**Pull request by dbudniko**
Accuracy thresholds and tolerance setup added in G-API module tests. 

```
buildworker:Mac=macosx-2
```

### CODE REVIEW

**Comment by dmatveev** [comment]
@dbudniko it seems build failed because of recently merged #12947 (and Sobel/sobel was that merge conflict)

File: modules/gapi/perf/common/gapi_core_perf_tests.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "gapi_core_perf_tests_inl.hpp"
```
Comment (dmatveev) [inline modules/gapi/perf/common/gapi_core_perf_tests.cpp]: Please also put a license header in this file and include "test_precomp.hpp" (before any other includes)

File: modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp [hunk 2]
```diff
@@ -0,0 +1,1777 @@
+#ifndef OPENCV_GAPI_CORE_PERF_TESTS_INL_HPP
```
Comment (dmatveev) [inline modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp]: Please put license header here as well

File: modules/gapi/perf/common/gapi_core_perf_tests.cpp (hunk 1 above)
Comment (dbudniko) [reply modules/gapi/perf/common/gapi_core_perf_tests.cpp]: perf_precomp.hpp added

File: modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp (hunk 2 above)
Comment (dbudniko) [reply modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp]: Done

**Comment by dmatveev** [comment]
@alalek thanks!
@dbudniko congrats!

**Pull request merged**